Taskfile.yml

```
@@ -0,0 +1,57 @@
+version: "3"
+
+includes:
+  lint: "lint-tasks.yml"
+  utils: "tools/yscope-dev-utils/taskfiles/utils.yml"
+
+vars:
+  G_BUILD_DIR: "{{.ROOT_DIR}}/target"
+  G_LINT_VENV_DIR: "{{.G_BUILD_DIR}}/lint-venv"
+
+tasks:
+  default:
+    deps: ["lint"]
+
+  clean:
+    cmds: ["rm -rf '{{.G_BUILD_DIR}}' '{{.G_LINT_VENV_DIR}}'"]
```

```suggestion
  clean: "rm -rf '{{.G_BUILD_DIR}}'"
```

* Since `G_LINT_VENV_DIR` is inside `G_BUILD_DIR`, we can just rely on deleting the build directory.
* We can use a shorthand since the task is just one command.